grunt/tasks/jest.js

```diff
@@ -77,7 +77,7 @@ function writeTempConfig(callback) {
 function run(done, configPath) {
   grunt.log.writeln('running jest (this may take a while)');
 
-  var args = ['--harmony', path.join('node_modules', 'jest-cli', 'bin', 'jest')];
+  var args = ['--harmony', path.join('node_modules', 'jest-cli', 'bin', 'jest'), '--runInBand'];
```

Comment:
cc @zpao @gaearon I'm gonna ship this with `--runInBand` – it should avoid the failures on travis you guys have been seeing. Let me know if you have any trouble with this.